docs/CONTRIBUTING.md

```diff
@@ -4,9 +4,16 @@ Thank you for contributing your idea to Fly-Pie! Here are some guidelines to hel
 
 ## Branching Guidelines
 
-The development of Fly-Pie follows a simplified version of **git-flow**: The `master` branch always contains stable code.
+The development of Fly-Pie follows a simplified version of **git-flow**:
+
+The `master` branch always contains stable code.
+
 New features and bug fixes are implemented in `feature/*` branches and are merged to `develop` once they are finished.
-When a new milestone is reached, the content of `develop` will be merged to `master` and a tag is created. In rare and critical cases, hotfixes for the latest release will be created in the `master` branch and merged back to `develop`.
+
+When a new version is about to get released, the content of `develop` will be merged to `release/vX` (`X` is the new version number) and the translators get notified.
+Once the translation work and last minute bugfixing is done, `release/vX` is merged into `master` and a tag is created to mark the release. The last step of the release process is to merge `master` back into `develop` in order to update the translations and bugfixes in the avtive development branch.
+
+In rare and critical cases, hotfixes for the latest release will be created in the `master` branch and merged back to `develop`.
```

Comment:
My suggested changes here are: itemized, "will be _branched_ to `release/vX`" and _active_ instead of _avtive_.

```suggestion
* When a new version is about to get released, the content of `develop` will be branched to `release/vX` (`X` is the new version number) and the translators get notified.
* Once the translation work and last minute bugfixing is done, `release/vX` is merged into `master` and a tag is created to mark the release. The last step of the release process is to merge `master` back into `develop` in order to update the translations and bugfixes in the active development branch.
* In rare and critical cases, hotfixes for the latest release will be created in the `master` branch and merged back to `develop`.
```

Reply:
Oh, didn't notice that merged/branched error at all :see_no_evil: Good point!